Map2/testdata/orders.js

```diff
@@ -1,2 +1,44 @@
-context={"gameID":366283,"variantID":1,"userID":129550,"memberID":1692910,"turn":11,"phase":"Builds","countryID":"2","tokenExpireTime":1625779489,"maxOrderID":"168362916","orderStatus":"Saved,Completed,Ready"};
```

Comment:
Revert this file

Reply:
This is simple indentation change for better visibility. I thought this can help anyone trying to understand the data.

Reply:
@hyukcodazen I understand, but we should not affect the legacy code unless we need to. Besides, all this data is in mockData.ts no? We have no need for these changes.